src/main/java/com/teragrep/rlo_06/RFC5424Parser.java

```diff
@@ -51,13 +51,20 @@
 
 public final class RFC5424Parser {
     private InputStream inputStream;
+    private final boolean LFT;
```

Comment:
please change variable name to LineFeedTermination

Reply:
Renamed variable to lineFeedTermination